package.json

```diff
@@ -3,9 +3,11 @@
   "version": "2.5.0",
```

Comment:
shouldn't there be a   "exports": {
    "import": "./index.js",
    "require": "./dist/index.js"
  },

or similar to make npm understand what to use when importing from cjs/esm?

Reply:
Indeed there should be! Fixed.